pants.ci.toml

```diff
@@ -12,7 +12,8 @@ memory_summary = true
 
 [test]
 use_coverage = true
-xml_dir = "dist/test-results/"
+report = true
+report_dir = "dist/test-results/"
```

Comment:
I don't think that it's actually necessary to set this anymore. It used to be for the Buildsense plugin to capture XML, but it is now generated and captured regardless of the report: #13025.